src/tools/cephfs/top/cephfs-top

```diff
@@ -1202,9 +1192,9 @@ if __name__ == '__main__':
                         help='Path to cluster configuration file')
     parser.add_argument('--selftest', dest='selftest', action='store_true',
                         help='Run in selftest mode')
-    parser.add_argument('-d', '--delay', nargs='?',
+    parser.add_argument('-d', '--delay', dest='delay', choices=range(1, 26),
```

Comment:
Does this mean values between 1 and 26 are allowed? (I haven't looked up what choices does).

Reply:
Values from 1 to 25 are allowed, as curses.halfdelay() limits to 255 .

Reply:
@neesingh-rh exactly